bootloader/bootloader.go

```diff
@@ -180,10 +205,10 @@ type TrustedAssetsBootloader interface {
 	// corresponding to the on-disk boot asset edition, followed by any
 	// extra arguments. The command line may be different when using a
 	// recovery bootloader.
```

Comment:
this comment needs some tweaks probably because of the use of CommandLineComponents and the fact those can have a full set of args now.

Reply:
Updated